README.md

```diff
@@ -1 +1,82 @@
-# HELLO_MICROVERSE-2023
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Table of Contents
+
+  * About the Project
+   - Built With
+
+  *Getting Started
+    - Prerequisites
+    - Setup
+    - Install
+    - Usage
+    - Run test
+    - Deployment
+
+  * Authors
+
+  * Contributing
+
+  * Show your support
+
+  * Acknowledgements
+
+  * License
+
+# HELLO MICROVERSE
+
+> My first project at microverse with a simple message "Hello Microverse"
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+To get a local copy up and running on your machine, follow these simple steps:
+Prerequisites
+In order to run this project you need:
+
+Setup
+Clone this repository to your desired folder:
+
+Install
+Install this project with:
+
+Usage
+To run the project, execute the following command:
+
+Run tests
+To run tests, run the following command:
+
+Deployment
+You can deploy this project using:
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: (https://github.com/3nityGitHub)
+- Twitter: (https://twitter.com/tbarijohnson)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] **[OPTIONAL]** Please consider removing this note only here to guide you on the template use. Removing this note will make your project look much more professionnal 😄